media/utils.py

```diff
@@ -15,6 +16,7 @@ class OEmbedData:
     object_id: str
     title: str
     thumbnail_url: str
+    track_id: int or None
```

Comment:
```suggestion
    track_id: typing.Optional[int]
```

Reply:
actually you never covert to `int` below and column is `CharField`:

```suggestion
    track_id: str
```

just default to empty string below since `CharField` is not nullable